addon/styles/addon.css

```diff
@@ -38,3 +38,32 @@
 .concept-scheme-multi-select-checkboxes__content {
   width: 100%;
 }
+
+/* SFB-115 */
+.CurrencyInput {
```

Comment:
Nitpick: I would use BEM casing to be consistent to how Appuniversum names things. `.currency-input`.

Reply:
Removed comment + use of BEM. Only not sure if I can leave the `input` inside the currency-input class. If we follow the BEM I think we should add a specific class or id to it